### PR TITLE
test(ref): test for resolving ref for server bindings object in Asyncapi3

### PR DESCRIPTION
**Pull request by cka121**
This PR adds unit tests for resolving reference for server bindings object in AsyncAPI 3 [#OSS-371](https://smartbear.atlassian.net/browse/OSS-371)

### My PR contains...

- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [ ] Bug fixes (non-breaking change which fixes an issue)
- [ ] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [X] are not breaking changes.

### Documentation
- [ ] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [ ] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [X] All new and existing tests passed.

### CODE REVIEW

**Review by robert-hebel-sb — APPROVED** [review]

File: packages/apidom-reference/test/dereference/strategies/asyncapi-3/server-bindings-object/index.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import { dereference } from '../../../../../src/index.ts';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const rootFixturePath = path.join(__dirname, 'fixtures');
+
+describe('dereference', function () {
+  context('strategies', function () {
+    context('asyncapi-3', function () {
+      context('Server Bindings Object', function () {
+        context('given in components/securitySchemes field', function () {
```
Comment (glowcloud) [inline packages/apidom-reference/test/dereference/strategies/asyncapi-3/server-bindings-object/index.ts]:
```suggestion
        context('given in server/bindings field', function () {
```

And we also need to add the test for `components/serverBindings`

**Review by glowcloud — APPROVED** [review]
